diff --git a/node_modules/@splinetool/react-spline/dist/Spline.d.ts b/node_modules/@splinetool/react-spline/dist/Spline.d.ts
index be4ef3a..8a399ac 100644
--- a/node_modules/@splinetool/react-spline/dist/Spline.d.ts
+++ b/node_modules/@splinetool/react-spline/dist/Spline.d.ts
@@ -15,6 +15,7 @@ export interface SplineProps extends Omit<React.HTMLAttributes<HTMLDivElement>,
     onSplineFollow?: (e: SplineEvent) => void;
     onSplineScroll?: (e: SplineEvent) => void;
     renderOnDemand?: boolean;
+    wasmPath?: string;
 }
 declare const Spline: import("react").ForwardRefExoticComponent<SplineProps & import("react").RefAttributes<HTMLDivElement>>;
 export default Spline;
diff --git a/node_modules/@splinetool/react-spline/dist/react-spline.js b/node_modules/@splinetool/react-spline/dist/react-spline.js
index 71ea9f8..a11da69 100644
--- a/node_modules/@splinetool/react-spline/dist/react-spline.js
+++ b/node_modules/@splinetool/react-spline/dist/react-spline.js
@@ -1,9 +1,9 @@
 "use client";
-import { jsx as f, jsxs as j, Fragment as z } from "react/jsx-runtime";
-import { forwardRef as D, useRef as U, useState as l, useEffect as F } from "react";
-import { Application as H } from "@splinetool/runtime";
-import I from "./ParentSize.js";
-const C = D(
+import { jsx as f, jsxs as z, Fragment as D } from "react/jsx-runtime";
+import { forwardRef as U, useRef as F, useState as l, useEffect as H } from "react";
+import { Application as I } from "@splinetool/runtime";
+import P from "./ParentSize.js";
+const G = U(
   ({
     scene: s,
     style: u,
@@ -18,13 +18,14 @@ const C = D(
     onSplineScroll: k,
     onLoad: r,
     renderOnDemand: E = !0,
-    children: g,
-    ...x
-  }, A) => {
-    const o = U(null), [c, a] = l(!0), [i, R] = l();
+    wasmPath: g,
+    children: x,
+    ...A
+  }, R) => {
+    const o = F(null), [c, a] = l(!0), [i, j] = l();
     if (i)
       throw i;
-    return F(() => {
+    return H(() => {
       a(!0);
       let e;
       const m = [
@@ -66,7 +67,10 @@ const C = D(
         }
       ];
       if (o.current) {
-        e = new H(o.current, { renderOnDemand: E });
+        e = new I(o.current, {
+          renderOnDemand: E,
+          wasmPath: g
+        });
         async function t() {
           await e.load(s);
           for (let n of m)
@@ -74,7 +78,7 @@ const C = D(
           a(!1), r == null || r(e);
         }
         t().catch((n) => {
-          R(n);
+          j(n);
         });
       }
       return () => {
@@ -83,14 +87,14 @@ const C = D(
         e.dispose();
       };
     }, [s]), /* @__PURE__ */ f(
-      I,
+      P,
       {
-        ref: A,
+        ref: R,
         parentSizeStyles: { overflow: "hidden", ...u },
         debounceTime: 50,
-        ...x,
-        children: () => /* @__PURE__ */ j(z, { children: [
-          c && g,
+        ...A,
+        children: () => /* @__PURE__ */ z(D, { children: [
+          c && x,
           /* @__PURE__ */ f(
             "canvas",
             {
@@ -106,5 +110,5 @@ const C = D(
   }
 );
 export {
-  C as default
+  G as default
 };
